src/SmartCommander/ViewModels/MainWindowViewModel.cs

```diff
@@ -152,44 +162,27 @@ public FilesPaneViewModel SecondPane
         {
             get
             {
-                if (LeftFileViewModel.IsSelected)
+                if (_SelectedPane == RightFileViewModel)
                 {
-                    return RightFileViewModel;
+                    return LeftFileViewModel;
                 }
-                else if (RightFileViewModel.IsSelected)
+                else
                 {
-                    return LeftFileViewModel;
+                    return RightFileViewModel;
                 }
-                throw new Exception(Resources.ErrorNoPane);
             }
         }
+        private FilesPaneViewModel _SelectedPane;
 
         public FilesPaneViewModel SelectedPane
```

Comment:
Maybe turn it to auto property?

Reply:
fixed

Reply:
Sorry, I don't mean that actually. My idea we can turn SelectedPane to auto property and get rid of _SelectedPane.

Reply:
I get It. 
_SelectedPane removed